pages/apply/[resident]/summary.tsx

```diff
@@ -27,6 +32,36 @@ const UserSummary = (): JSX.Element => {
 
   const baseHref = `/apply/${currentResident.person?.id}`;
   const returnHref = '/apply/overview';
+  const application = useAppSelector((store) => store.application);
+
+  const applicants = useAppSelector((store) =>
+    [store.application.mainApplicant, store.application.otherMembers]
+      .filter((v): v is Applicant | Applicant[] => v !== undefined)
+      .flat()
+  );
+
+  const eligibilityMap = useMemo(
+    () =>
+      new Map(
+        applicants.map((applicant) => [applicant, checkEligible(applicant)[0]])
+      ),
+    [applicants]
+  );
+
+  const mainApplicant = applicants
+    .filter((app) => app === application.mainApplicant)
+    .map((ma) => ma)
+    .reduce((map) => map);
+
+  const onConfirmData = () => {
+    const isMainApplicantEligible = eligibilityMap.get(mainApplicant);
```

Comment:
I think we need to check if all members involved in the application are eligible, not just the main applicant.